docs/adrs/003-Retry_endpoint_design.md

```diff
@@ -100,7 +103,6 @@ A: Not necessarily relevant to this endpoint, more of a run_job_batch endpoint q
 Priority descending
 * Non blocking job submission for submitting multiple jobs, possibly via using `run_job_batch` (requires refactor of run_job_batch)
 * One single submission to HTCondor instead of multiple job submission ()
-* Ability to gracefully handle jobs with children (may require refactoring models)
 * Prevent multiple in-flight retries to prevent the user from wasting their own resources (and the queues resources)
 * Add retry_count to retried jobs as well to aid in more book-keeping in a new field called `retry_number`
```

Comment:
I think that anything that might change the API needs to be top priority so the FE folks don't code against something that's going to change. I think the only issue that fits that description is adding the retry number to the child jobs if we decide to do that.

The remainder of the stuff is more loosy goosy but IMO dealing with potential data inconsistency should be near the top.

Reply:
> dealing with potential data inconsistency should be near the top.

💯